src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -67,4 +69,24 @@ public boolean equals(Object other) {
         return id.getText().equals(card.id.getText())
                 && person.equals(card.person);
     }
+    /**
+      * Creates tag labels for {@code person}.
+      */
+    private void tagsInitialise(Person person) {
+        person.getTags().forEach(tag -> {
+            Label labelTags = new Label(tag.tagName);
+            labelTags.getStyleClass().add(getColorForTags(tag.tagName));
+            tags.getChildren().add(labelTags);
+        });
+    }
+    /**
+    * Returns color style for {@code tagName}'s label.
+    */
+    private String getColorForTags(String tagName) {
+        /**
+         * we use the hash code of the tag name to generate a random color, so that the color remain consistent
```

Comment:
Could you rephrase this comment to describe what the code does and not what the developer does?